### PR TITLE
Lo 9 create CSS file

### PR DESCRIPTION
**Pull request by yoyi1073**
CSS file

### CODE REVIEW

File: resources/views/leads.blade.php [hunk 1]
```diff
@@ -1,63 +1,104 @@
 <x-layout>
-    <h3>Registro</h3>
+        <h1>¡Regístrate en nuestra plataforma!</h1>
+        <p>Bienvenido a Kapital, completa el registro</p>
```
Comment (zahirolamadrid) [inline resources/views/leads.blade.php]: Remove Kapital referencies

File: resources/views/leads.blade.php [hunk 2]
```diff
@@ -1,63 +1,104 @@
...
-            <input type="text" id="last-name" name="last-name" placeholder="Primer apellido" required>
+            <input id="privacy" type="checkbox" name="privacy" checked>
+            <label for="privacy" class="terms">He leído y acepto el <a href=""><strong>Aviso de Privacidad</strong></a></label>
         </div>
         <br>
         <div>
-            <label for="second-last-name">Segundo apellido:</label>
-            <input type="text" id="second-last-name" name="second-last-name" placeholder="Segundo apellido" required>
+            <input id="declaration" type="checkbox" name="declaration">
+            <label for="declaration" class="terms"><a href=""><strong>Declaraciones</strong></a> de aceptación de prestación de servicios y/o productos con Corporación Nomi Fin, S.A. de C.V., SOFOM, E.N.R.</label>
```
Comment (zahirolamadrid) [inline resources/views/leads.blade.php]: Remove business references

File: resources/views/leads.blade.php (hunk 1 above)
Comment (yoyi1073) [reply resources/views/leads.blade.php]: Done!

File: resources/views/leads.blade.php [hunk 3]
```diff
@@ -1,63 +1,104 @@
...
+                <input type="text" id="last-name" name="last-name" placeholder="Primer apellido*" required>
+            </div>
+            <br>
+            <div>
+                <label class="info" for="second-last-name">Segundo apellido:</label>
+                <input type="text" id="second-last-name" name="second-last-name" placeholder="Segundo apellido*" required>
+            </div>
+            <br>
+            <div>
+                <label class="info" for="rfc">RFC</label>
```
Comment (Diana9218) [inline resources/views/leads.blade.php]: After RFC put a colon

File: resources/views/leads.blade.php (hunk 2 above)
Comment (yoyi1073) [reply resources/views/leads.blade.php]: Done!

File: resources/views/leads.blade.php [hunk 4]
```diff
@@ -1,63 +1,104 @@
...
         <br>
         <div>
-            <label for="second-last-name">Segundo apellido:</label>
-            <input type="text" id="second-last-name" name="second-last-name" placeholder="Segundo apellido" required>
+            <input id="declaration" type="checkbox" name="declaration">
+            <label for="declaration" class="terms"><a href=""><strong>Declaraciones</strong></a> de aceptación de prestación de servicios y/o productos con Corporación Nomi Fin, S.A. de C.V., SOFOM, E.N.R.</label>
         </div>
         <br>
-        <div>
-            <label for="rfc">RFC</label>
```
Comment (Diana9218) [inline resources/views/leads.blade.php:42]: Missing colon

**Review by zahirolamadrid — APPROVED** [review]

**Review by Diana9218 — APPROVED** [review]

**Review by Luismgi — APPROVED** [review]

**Review by Luismgi — APPROVED** [review]

**Pull request merged**